libdumpalloc/CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ target_link_libraries ( dumpalloc-python "${PYTHON_LIBRARIES}" )
 target_include_directories ( dumpalloc-python PRIVATE "${PYTHON_INCLUDE_DIRS}" )
 endif()
 
-add_subdirectory ( manual-test-programs )
+#add_subdirectory ( manual-test-programs )
```

Comment:
Yes. manual-test-programs doesn't exist there, so this throws an error.

maybe you have some local files that aren't checked in?